Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocCompoundColumnRendererTest.cs

```diff
@@ -176,7 +176,7 @@ public void RenderEnforcedWidthCell ()
       Html.AssertStyleAttribute(cropSpan, "overflow", "hidden");
       Html.AssertStyleAttribute(cropSpan, "white-space", "nowrap");
 
-      var span = Html.GetAssertedChildElement(cropSpan, "span", 0);
```

Comment:
Here is also a parent element `cropSpan` referenced, which indicates we now try to nest a `div` inside a `span` which is considered a big no-no. Checking further up in the test, `cropSpan` is indeed a `span` so there is a change still missing.